packages/gatsby-kontent-components/tests/rich-text-emelemt.spec.tsx

```diff
@@ -156,9 +177,9 @@ describe('<RichTextElement/>', () => {
   it('Resolve linked items', () => {
     const testRenderer = TestRenderer.create(
       <RichTextElement
-        value={sampleComplexValue}
+        value={"<p>This is Ondřej Chrastina - Developer Advocate with <a href=\"https://kontent.ai\" data-new-window=\"true\" target=\"_blank\" rel=\"noopener noreferrer\">Kentico Kontent</a>.</p>\n<figure data-asset-id=\"d32b8ad5-0cf4-47a8-8b53-ed4a1e80dc88\" data-image-id=\"d32b8ad5-0cf4-47a8-8b53-ed4a1e80dc88\"><img src=\"https://assets-us-01.kc-usercontent.com:443/0f6785d7-8214-000f-62b3-d9580d692b59/44299668-b37b-4224-a115-1fd66f7d7b36/Yprofile.jpg\" data-asset-id=\"d32b8ad5-0cf4-47a8-8b53-ed4a1e80dc88\" data-image-id=\"d32b8ad5-0cf4-47a8-8b53-ed4a1e80dc88\" alt=\"\"></figure>\n<p>He likes to do web sites. This is his latest project:</p>\n<object type=\"application/kenticocloud\" data-type=\"item\" data-rel=\"link\" data-codename=\"ondrej_chrastina_tech\"></object>\n<p><br>\nHe also likes OSS. This is his latest repository:</p>\n<object type=\"application/kenticocloud\" data-type=\"item\" data-rel=\"link\" data-codename=\"simply007_kontent_gatsby_benchmark\"></object>\n<p><br>\nOn some projects, he was cooperating with <a data-item-id=\"2b805947-7ca5-4e6a-baa5-734a91f3cfa2\" href=\"\">John Doe</a>.</p>\n<p>You could take a look at their&nbsp;<a href=\"https://google.com\" title=\"sample link\">latest project</a>.</p>\n<p>Check out also this interesting repository</p>\n<object type=\"application/kenticocloud\" data-type=\"item\" data-rel=\"component\" data-codename=\"n5b76e971_4709_01f2_e8ea_36a9716d7b42\"></object>"}
         linkedItems={linkedItems}
-        resolveLinkedItem={(linkedItem): JSX.Element => {
+        resolveLinkedItem={(linkedItem,domNode): JSX.Element => {
           return <pre>{JSON.stringify(linkedItem, undefined, 2)}</pre>;

```

Comment:
use the `domNode` value in the return statement, so that is it verified, it is being used. Or at least place the assertion in the resolveLinkedItem for domNode.

Reply:
Check the `domNode` is not null as an assert and assert the inner HTML.

Reply:
I need help with this